content/en/docs/reference/glossary/multi-cluster.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Multi-Cluster
+---
+
+Multi-cluster is a deployment model that consists of a
+[mesh](/docs/reference/glossary/#service-mesh) with multiple
```

Comment:
```suggestion
single [service mesh](/docs/reference/glossary/#service-mesh) with multiple
```

Reply:
`a mesh` is already singular. I avoided `service` to keep the definition focused on the cluster level.